app/views/katello/api/v2/subscriptions/permissions.rabl

```diff
@@ -0,0 +1,11 @@
+collection @subscriptions
+
+if params.key?(:include_permissions)
+  user = User.current
```

Comment:
Why not use [`current_user`](https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/api_controller.rb#L14)?